docs/topics/vscode-extension-resolving-issues.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * docs/vs-code-extension-guide/master.adoc
+
+[id='vscode-extension-resolving-issues_{context}']
+= Resolving issues
+
+You can resolve issues by using a Quick Fix (automatic code replacement) or by editing the code.
+
+.Procedure
+
+. Resolve issues detected by the {ProductShortName} extension by performing one of the following actions:
+
+* If the issue displays the Quick Fix icon (image:vs_optional.png[Optional or Quick Fix]), right-click the issue and select *Preview Quick Fix*. To accept the suggested fix, right-click the issue and select *Apply Quick Fix.*
+* If the issue has any other icon or you decide not to use all suggested Quick Fixes, right-click the issue, select *Open Code* to open the file involved, make any changes needed to the code, and then save the file.
+
+. Optional: After you fix an issue, you can right-click it and select *Mark as Complete*. The extension displays the Resolved icon (image:vs_resolved.png[Resolved]) next to the issue. This has no effect on the code.
```

Comment:
There seems to be a contradiction here: "After you fix an issue" means that the code has changed. But marking it has complete does not change the code. 

It seems to me that the "Mark as Complete" is probably not connected to whether or not the user has fixed an issue. My guess is that the user can mark it as complete because either the code is fixed or the code needs no fix. In that case, "After you fix an issue" is superfluous.

So check the usage of this feature. Also, why does marking as complete result in a Resolved icon? Maybe the icon should be called "Complete", not "Resolved".